contracts/bonding/libraries/EarningsPool.sol

```diff
@@ -30,7 +30,7 @@ library EarningsPool {
     }
 
     /**
-     * @dev Sets transcoderRewardCut and transcoderFeeshare for an EarningsPool
+     * @notice Sets transcoderRewardCut and transcoderFeeshare for an EarningsPool
```

Comment:
Should these functions use the `@dev` tag since they are internal and not user facing? Same comment applies to the rest of the file.

Reply:
fixed in 8d0bb3e